src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp

```diff
@@ -158,28 +158,28 @@
                     <label>Present: </label>
                     <br/>
                     <label for="presentPoints">
-                        <form:input type = "text" path ="presentPoints" id = "presentPoints" placeholder="100" size="7"/>
+                        <form:input type = "text" path ="presentPoints" id = "presentPoints" size="7" onkeyup="toggleButton()"/>
                     </label>
                 </div>
                 <div class="col-md-2 col-md-offset-0">
                     <label>Tardy: </label>
                     <br/>
                     <label for="tardyPoints">
-                        <form:input type = "text" path ="tardyPoints" id = "tardyPoints" placeholder="0" size="7"/>
+                        <form:input type = "text" path ="tardyPoints" id = "tardyPoints" size="7" onkeyup="toggleButton()"/>
                     </label>
                 </div>
                 <div class="col-md-2 col-md-offset-0">
                     <label>Absent: </label>
                     <br/>
                     <label for="absentPoints">
-                        <form:input type = "text" path ="absentPoints" id = "absentPoints" placeholder="0" size="7"/>
+                        <form:input type = "text" path ="absentPoints" id = "absentPoints" size="7" onkeyup="toggleButton()"/>
                     </label>
                 </div>
                 <div class="col-md-2 col-md-offset-0">
                     <label>Excused: </label>
                     <br/>
                     <label for="excusedPoints">
-                        <form:input type = "text" path ="excusedPoints" id = "excusedPoints" placeholder="0" size="7"/>
+                        <form:input type = "text" path ="excusedPoints" id = "excusedPoints" size="7" onkeyup="toggleButton()"/>
```

Comment:
you could also probably replace the "onkeyup='toggleButton()'" with a jquery listener to avoid the having to put it in all the fields like:
$("#excusedPoints, #absentPoints, .. [other ids of the fields you want]").keyup(function(){ toggleButton();});
or create an element class to add to the fields that you want to have the onkeyup listener like "updatable-field"
then the jquery listener would look something like this:
$(".updateable-field").keyup(function(){ toggleButton();});